### PR TITLE
[new release] decoders, decoders-ezjsonm and decoders-yojson (0.1.0)

### PR DESCRIPTION
**Pull request by mattjbray**
Elm-inspired decoders for Ocaml

- Project page: <a href="https://github.com/mattjbray/ocaml-decoders">https://github.com/mattjbray/ocaml-decoders</a>
- Documentation: <a href="https://mattjbray.github.io/ocaml-decoders/decoders">https://mattjbray.github.io/ocaml-decoders/decoders</a>

##### CHANGES:

Initial release.

### CODE REVIEW

File: packages/decoders/decoders.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+After updating CHANGES.md:
+
+```
+npm version <newversion>
+dune-release tag
+dune-release bistro -p decoders,decoders-ezjsonm,decoders-yojson
+```"""
+maintainer: "Matt Bray <matt@aestheticintegration.com>"
+authors: "Matt Bray <matt@aestheticintegration.com>"
+license: "LICENSE.md"
```
Comment (kit-ty-kate) [inline packages/decoders/decoders.0.1.0/opam]: mmh this is not what this field is about. It should state (roughly if you need to) what the name of the license si

**Comment by kit-ty-kate** [comment]
```
#=== ERROR while compiling decoders.0.1.0 =====================================#
# context     2.0.0 | linux/x86_64 | ocaml-base-compiler.4.02.3 | file:///home/travis/build/ocaml/opam-repository
# path        ~/.opam/4.02.3/.opam-switch/build/decoders.0.1.0
# command     ~/.opam/opam-init/hooks/sandbox.sh build jbuilder build -p decoders -j 1
# exit-code   1
# env-file    ~/.opam/log/decoders-14869-d0f245.env
# output-file ~/.opam/log/decoders-14869-d0f245.out
### output ###
#       ocamlc src/.decoders.objs/decoders__Decode.{cmi,cmti} (exit 2)
# (cd _build/default && /home/travis/.opam/4.02.3/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.decoders.objs -I src/.decoders.objs/.private -I /home/travis/.opam/4.02.3/lib/containers -I /home/travis/.opam/4.02.3/lib/containers/monomorphic -I /home/travis/.opam/4.02.3/lib/result -I /home/travis/.opam/4.02.3/lib/uchar -no-alias-deps -open Decoders__ -o src/.decoders.objs/decoders__Decode.cmi -c -[...]
# File "src/decode.mli", line 10, characters 54-87:
# Error: Unbound type constructor result
#       ocamlc src/.decoders.objs/decoders__Util.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/travis/.opam/4.02.3/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.decoders.objs -I src/.decoders.objs/.private -I /home/travis/.opam/4.02.3/lib/containers -I /home/travis/.opam/4.02.3/lib/containers/monomorphic -I /home/travis/.opam/4.02.3/lib/result -I /home/travis/.opam/4.02.3/lib/uchar -no-alias-deps -open Decoders__ -o src/.decoders.objs/decoders__Util.cmo -c -im[...]
# File "src/util.ml", line 40, characters 25-56:
# Error: Unbound type constructor Pervasives.result
```

**Comment by mattjbray** [comment]
Thanks. What's the best workflow to address these?  I'm thinking:

1. Close this PR 
2. Delete the tag and release tarball from GitHub
3. Fix the issues
4. Use `dune-release` to open a new PR here

**Comment by mseri** [comment]
For a version 0.1 this seems a reasonable way to proceed, and the latest dune-release should do most of the dirty release job for you

**Pull request closed without merging**